Makefile

```diff
@@ -57,7 +57,9 @@ login:
 
 push: build login
 	docker push $(REPO):$(VERSION)
-	docker push $(REPO):latest
+	if [[ ${TRAVIS_TAG} =~ ^[0-9]+(\.[0-9]+)+(\.[0-9]+)$ ]]; then
```

Comment:
You don't need to create capture groups. Check out the regex pattern we are already using in the code: https://github.com/pravega/pravega-operator/blob/master/pkg/util/pravegacluster.go#L29

Reply:
Got it, thanks!